packages/plugin-devtools/src/components/TreeComponent.ts

```diff
@@ -126,7 +126,7 @@ export class TreeComponent extends OwlComponent<NodeProps> {
      * @param node
      */
     isFormat(node: VNode, formatName: string): boolean {
-        return node.is(InlineNode) && !!node.formats.find(format => format.name === formatName);
+        return node.is(InlineNode) && !!node.modifiers.find(format => format.name === formatName);
```

Comment:
Nothing changed on that level : we need to be able to identify them in an XML file... I could also use the constructor name instead of course. But `format.name` is also used to nicely represent the format in `PathComponent` so I have it anyway.